report/sections/titlepage.tex

```diff
@@ -0,0 +1,22 @@
+\begin{titlepage}
+  
+  \begin{center}
+    \begin{LARGE}
+      Department of Electrical and Computer Engineering \\
+      University of Victoria \\
+      ELEC 483 - Digital Video Processing \\[1cm]
+      \textsc{Invisible watermarks}
```

Comment:
The W in watermarks can be capitalized here in the title. (Stylistic choice of course).